custom/icds/messaging/indicators.py

```diff
@@ -397,6 +383,34 @@ def get_messages(self, language_code=None):
         return messages
 
 
+def get_awws_with_old_vhnd_date(domain, user_ids):
+    return [_id for _id in user_ids if _id not in precompute_awws_in_vhnd_timeframe(domain)]
+
+
+@icds_quickcache(timeout=12 * 60 * 60, memoize_timeout=12 * 60 * 60)
```

Comment:
I think you also want to set `session_function=None` otherwise it will only be memoized for the duration of a single task

Reply:
Thanks